google-cloud-core/src/test/java/com/google/cloud/TimestampTest.java

```diff
@@ -240,7 +240,7 @@ public void testToString() {
   }
 
   @Test
-  public void parseTimestamp() {
+  public void parseTimestampWithTimeZoneOffset() {
     assertThat(Timestamp.parseTimestamp("0001-01-01T00:00:00Z")).isEqualTo(Timestamp.MIN_VALUE);
     assertThat(Timestamp.parseTimestamp("9999-12-31T23:59:59.999999999Z"))
         .isEqualTo(Timestamp.MAX_VALUE);
```

Comment:
I don't see a string with time zone offset.

Reply:
I meant adding a timezone offset to the test. Actually having `parseTimestampWithTimeZoneOffset` was a good idea, but that test should use strings "WithTimeZoneOffset".

Reply:
revert back this change and add new test called `parseTimestampWithTimeZoneOffset ` same like 
`parseTimestampWithoutTimeZoneOffset`